src/test/scala/com.lightbend.paradox/unidoc/MarkdownBaseSpec.scala

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+
```

Comment:
This is a copy of [the same file in the paradox repo](https://github.com/lightbend/paradox/blob/master/core/src/test/scala/com/lightbend/paradox/markdown/MarkdownBaseSpec.scala), as there is no jar published to reuse it.

Reply:
Looks like the time has come for paradox-testkit. :)

Reply:
I can try to split out the paradox-testkit jar later, but I think in this PR, copying MarkdownBaseSpec here is easier.

Reply:
Yup. Lets have a copy for now.